packages/hash/frontend/src/blocks/blocksMeta.tsx

```diff
@@ -22,7 +22,10 @@ export const BlocksMetaProvider: React.FC<{ value: BlocksMetaMap }> = ({
   value: initialValue,
   children,
 }) => {
-  const [value, setValue] = useLocalstorageState("blocks-meta", initialValue);
+  const [value, setValue] = useLocalstorageState(
+    "hash-workspace-blocks-meta",
```

Comment:
This change was made to uniquely identify `hash-workspace` in localStorage for localhost development and prevent collisions.